CMakeLists.txt

```diff
@@ -171,6 +171,36 @@ if(ENABLE_NLS)
   endif()
 endif()
 
+option(ENABLE_H264 "Enable H.264 RFB encoding" ON)
+if(ENABLE_H264)
+  if(WIN32)
+    add_definitions("-DHAVE_H264")
+    set(H264_LIBS "WIN")  # may be LIBAV in the future
+    set(H264_LIBRARIES ole32 mfplat mfuuid wmcodecdspuuid)
+  else()
+    check_include_files(libavcodec/avcodec.h HAVE_AVCODEC_H)
+    check_include_files(libavutil/avutil.h HAVE_AVUTIL_H)
+    check_include_files(libswscale/swscale.h HAVE_SWSCALE_H)
+    if (HAVE_AVCODEC_H AND HAVE_AVUTIL_H AND HAVE_SWSCALE_H)
+      find_path(AVCODEC_INCLUDE_DIR libavcodec/avcodec.h)
+      find_library(AVCODEC_LIBRARY avcodec)
+      find_path(AVUTIL_INCLUDE_DIR libavutil/avutil.h)
+      find_library(AVUTIL_LIBRARY avutil)
+      find_path(SWSCALE_INCLUDE_DIR libswscale/swscale.h)
+      find_library(SWSCALE_LIBRARY swscale)
+      include_directories(${AVCODEC_INCLUDE_DIR} ${AVUTIL_INCLUDE_DIR} ${SWSCALE_INCLUDE_DIR})
+      set(H264_LIBRARIES ${AVCODEC_LIBRARY} ${AVUTIL_LIBRARY} ${SWSCALE_LIBRARY})
+      add_definitions("-D__STDC_CONSTANT_MACROS")
```

Comment:
`__STDC_CONSTANT_MACROS` may not be necessary? Per [this issue](https://sourceware.org/bugzilla/show_bug.cgi?id=15366).

Reply:
RHEL 7 is glibc 2.17, so this could still be relevant if people want this on older systems.